_config.yml

```diff
@@ -4,7 +4,7 @@
 # Make our URLs look nice. We're not a blog with dated posts, we just want the title.
 permalink:     /:title.html
 
-gems:
+plugins:
```

Comment:
I changed it back to `gems` but `htmlproofer` still returns 2410 errors (same as with `plugins`), so I am not sure that this line is the reason for the issue.